Android/toonDra/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.2.0-beta3'
+    classpath 'com.android.tools.build:gradle:2.2.0'
 
     // NOTE: Do not place your application dependencies here; they belong
     // in the individual module build.gradle files
```

Comment:
This file change is also not part of your logical unit. Please make this a separate pull request.